src/main/java/net/dries007/tfc/types/DefaultRecipes.java

```diff
@@ -319,7 +319,7 @@ public static void onRegisterHeatRecipeEvent(RegistryEvent.Register<HeatRecipe>
             HeatRecipe.destroy(IIngredient.of(ItemFoodTFC.get(Food.COOKED_VENISON)), 480).setRegistryName("burned_venison"),
 
             // Egg
-            HeatRecipe.destroy(IIngredient.of(Items.EGG), 480).setRegistryName("burned_egg")
+            HeatRecipe.destroy(IIngredient.of(ItemFoodTFC.get(Food.COOKED_EGG)), 480).setRegistryName("burned_egg")
```

Comment:
This destroys cooked eggs, yes, but the issue was that eggs aren't able to be cooked in the first place? See L297: 
```java
new HeatRecipeSimple(IIngredient.of(Items.EGG), new ItemStack(ItemFoodTFC.get(Food.COOKED_EGG)), 200, 480).setRegistryName("cooked_egg"),
```
which should already be destroying eggs. I don't see how this fixes this issue (but it may just be I'm confused)

Reply:
The issue is rather simple:
https://github.com/TerraFirmaCraft/TerraFirmaCraft/blob/8519ab95386ffd3d3ee5647efb270e50e7ff43c4/src/main/java/net/dries007/tfc/api/recipes/heat/HeatRecipe.java#L41-L44
returns the first valid found recipe, which was returning the destruction recipe. Just looking at the above lines you will find that the other cooking is registered by Ingredient -> Cooked, and then Cooked -> Destroy recipe (at a higher temp).

Reply:
Perfect, thanks